README.md

```diff
@@ -24,6 +25,9 @@ However, to use all features of GraftM a few extra binary applications are requi
 To create new GraftM packages, you'll also need
 * FastTreeMP (http://www.microbesonline.org/fasttree/)
 
+#### Docker images
+Versions of graftM on pip now have matching docker images as of GraftM v0.9.2. GraftM docker images are portable filesystems that contain the graftM code and all its dependancies, allowing GraftM to be run on any platform with docker installed. Details on how to download and run a GraftM docker image can be found on the [graftm-docker](https://github.com/geronimp/graftM-docker) GitHub page or the [docker hub page](https://hub.docker.com/u/geronimp/).
```

Comment:
changing that slightly, `The docker images contain GraftM and all its python and non-python dependencies, allowing GraftM to be run on any platform with docker installed`

better? Technically they are more than filesystems, they are containers.